crates/node/builder/Cargo.toml

```diff
@@ -74,3 +74,13 @@ backon.workspace = true
 
 [dev-dependencies]
 tempfile.workspace = true
+
+[features]
+optimism = [
+    "reth-primitives/optimism",
+    "reth-rpc/optimism",
+    "reth-provider/optimism",
+    "reth-beacon-consensus/optimism",
+    "reth-blockchain-tree/optimism",
+    "reth-node-core/optimism",
+]
```

Comment:
I don't think this is required?

we want to keep this crate feature free and instead push all things down to the `node` crates like you did with consensus

Reply:
Good catch - this was left over from my original implementation